inst/lintr/linters.csv

```diff
@@ -62,3 +62,5 @@ undesirable_function_linter,style efficiency configurable robustness best_practi
 undesirable_operator_linter,style efficiency configurable robustness best_practices
 unneeded_concatenation_linter,style readability efficiency
 vector_logic_linter,default efficiency best_practices
+yoda_test_linter,package_development best_practices readability
+
```

Comment:
Is this an extra newline? Can't see this in GitHub.

Reply:
yes! nice catch